plugins/workbench-resources/src/connect.ts

```diff
@@ -192,6 +192,7 @@ export async function connect (title: string): Promise<Client | undefined> {
   // Update window title
   document.title = [ws, title].filter((it) => it).join(' - ')
   await setClient(_client)
+  clientSet = true
```

Comment:
Why we need it?

Reply:
To not refresh if was not client yet, it was missed in if.